src/foolscap/test/test_appserver.py

```diff
@@ -135,15 +135,15 @@ def test_create(self):
         serverdir = os.path.join(basedir, "fl")
         d = self.run_cli("create", "--location", "localhost:1234", serverdir)
         def _check((rc,out,err)):
-            self.failUnlessEqual(rc, 0)
-            self.failUnless(os.path.isdir(serverdir))
+            self.assertEqual(rc, 0)
+            self.assertTrue(os.path.isdir(serverdir))
             # check that the directory is group/world-inaccessible, even on
             # windows where those concepts are pretty fuzzy. Do this by
             # making sure the mode doesn't change when we chmod it again.
             mode1 = os.stat(serverdir).st_mode
             os.chmod(serverdir, 0700)
```

Comment:
I'm surprised 2to3 didn't catch this.. it's not in the diff, but that (and any other octal literal) should be written as `0o700`.

Reply:
Only the assertion changes were done using 2to3. I try to avoid automated changes otherwise. Will fix this.

Reply:
Yeah, I understand trying to keep the size of the diff down, but `0700` is invalid syntax under py3. It could be split off into another PR, but it _is_ necessary for having 2/3 compat.

Reply:
Thought I fixed all chmod octal values. Good catch. Fixed it.